src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {
+        storage.addFruit(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
The `BalanceOperation` class seems to be doing its job, but ensure that it follows the SOLID principles. Specifically, the Single Responsibility Principle: the class should only be responsible for handling the 'balance' operation and not directly interact with the `Storage`. Consider abstracting the storage interaction to another layer or service.